packages/documentation/src/stories/foundation/zindex/zindex.stories.mdx

```diff
@@ -0,0 +1,30 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta
+  title="Foundations/Z-Index"
```

Comment:
I'm not happy with that title, but we can discuss that when the documentation is written.

Reply:
I chaneged it to "elevation". Hope you can agree with that?